pageserver/src/layered_repository.rs

```diff
@@ -796,18 +796,18 @@ impl Timeline for LayeredTimeline {
             lsn, latest_gc_cutoff_lsn
         );
 
-        let seg = SegmentTag::from_blknum(rel, blknum);
+        let (seg, seg_blknum) = SegmentTag::from_blknum(rel, blknum);
```

Comment:
I think this conversion is premature. We go from true blk # -> local blk # here. Then in materialize page, we go backwards. I think the conversion

Reply:
Hmm, it's true that it looks weird when you look at this caller. However, looking at the call from ImageLayer::create_from_src, if we changed it, it would look equally weird.